WordPress/Classes/ViewRelated/Blog/Blog Details/PostsCardViewController.swift

```diff
@@ -0,0 +1,36 @@
+import UIKit
+
+/// Render a small list of posts for a given blog and post status (drafts or scheduled)
+///
+/// This class handles showing posts from the database, syncing and interacting with them
+///
+@objc class PostsCardViewController: UIViewController {
+    @objc var blog: Blog?
+
+    private let postsTableView = IntrinsicTableView()
+
+    private var viewModel: PostsCardViewModel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureView()
+        viewModel = PostsCardViewModel(tableView: postsTableView, blog: blog!)
+        viewModel.viewDidLoad()
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        postsTableView.dataSource = viewModel
+        viewModel.refresh()
+    }
+}
+
+private extension PostsCardViewController {
+    func configureView() {
+        view.addSubview(postsTableView)
+        postsTableView.translatesAutoresizingMaskIntoConstraints = false
+        postsTableView.pinSubviewToAllEdges(view)
```

Comment:
```suggestion
        view.pinSubviewToAllEdges(postsTableView)
```

[nit] This technically has the same result, but the param should be the subview being pinned to the superview
```
    /// Adds constraints that pin a subview to self with zero insets.
    ///
    /// - Parameter subview: a subview to be pinned to self.
    @objc public func pinSubviewToAllEdges(_ subview: UIView) {
        pinSubviewToAllEdges(subview, insets: .zero)
    }
```

Reply:
Done in b44161ae2c73cf8d720ab4197c535575bce1f2e3